ansible/roles/iroha-docker/templates/docker-compose.yml.j2

```
@@ -52,7 +52,7 @@ services:
 {% endif %}
     restart: always
 {% if iroha_docker_commands is defined %}
-    command: {{ iroha_docker_commands | to_yaml }}
+    command: {{ iroha_docker_commands | to_json }}
```

Add comment about it
```suggestion
    # Only JSON notation for commands can work in docker-compose
    command: {{ iroha_docker_commands | to_json }}
```